src/test/java/de/tum/in/www1/artemis/util/GitUtilService.java

```diff
@@ -286,4 +287,14 @@ public void writeEmptyJsonFileToPath(Path path) throws Exception {
             writer.write(fileContent);
         }
     }
+
+    /**
+     * Creates a CommitCommand and sets signing to false.
+     *
+     * @param git Git Repository Object.
+     * @return CommitCommand with signing set to false.
+     */
+    public CommitCommand commit(Git git) {
```

Comment:
I noticed the same method in two classes. Is there a specific reason for this? If they're doing the same thing, maybe we could put this in one of them.

Reply:
I found that the tests use GitService and GitUtilService also has issues when no Signing Service is available. So I added it there as well.